### PR TITLE
Updated travis configuration using industrial_ci

### PR DESCRIPTION
**Pull request by progtologist**
Please update the recipients accordingly.
Also refers to issue #61

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,35 @@
...
-python:
-  - "2.7"
+# This config file for Travis CI utilizes ros-industrial/industrial_ci package.
+# For more info for the package, see https://github.com/ros-industrial/industrial_ci/blob/master/README.rst
+sudo: required 
+dist: trusty 
+services:
+  - docker
+language: generic 
 compiler:
```
Comment (ipa-mdl) [inline .travis.yml]: the compiler block can be removed if for `language: generic`.

File: .travis.yml [hunk 2]
```diff
@@ -1,48 +1,35 @@
...
-  # Install dependencies for source repos
-  - rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
-before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
-  - source /opt/ros/$ROS_DISTRO/setup.bash  
-script: # All commands must exit with code 0 on success. Anything else is considered failure.
-  - catkin_make -j2
-
+      - enrique.fernandez.perdomo@gmail.com
+      - bence.magyar.robotics@gmail.com    
+      - gm130s@gmail.com
```
Comment (ipa-mdl) [inline .travis.yml]: I guess that @130s does not want to get notified

File: .travis.yml [hunk 3]
```diff
@@ -1,48 +1,35 @@
...
+    - ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO="kinetic"   UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall
+    - ROS_DISTRO="jade"      ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="jade"      ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO="jade"      UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall
+install:
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+script:
+  - source .ci_config/travis.sh
```
Comment (ipa-mdl) [inline .travis.yml]: sourcing is not recommended anymore, just call it `- .ci_config/travis.sh`

File: .travis.yml [hunk 4]
```diff
@@ -1,48 +1,35 @@
-# Travis Continuous Integration Configuration File For ROS Control Projects
-# Author: Dave Coleman
-language:
-  - cpp
-  - python
-python:
-  - "2.7"
+# This config file for Travis CI utilizes ros-industrial/industrial_ci package.
```
Comment (ipa-mdl) [inline .travis.yml]: Please adapt to the actual repository.

File: .travis.yml [hunk 5]
```diff
@@ -1,48 +1,35 @@
...
+      - gm130s@gmail.com
+env:
+  global:
+    - CATKIN_PARALLEL_TEST_JOBS=-p1
+    - ROS_PARALLEL_TEST_JOBS=-j1
+  matrix:
+    - ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO="kinetic"   UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall
+    - ROS_DISTRO="jade"      ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
```
Comment (ipa-mdl) [inline .travis.yml]: Lunar support:
`- ROS_DISTRO="lunar"     ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu`
or (shorter)
`- ROS_DISTRO="lunar"     ROS_REPO=ros-shadow-fixed`

File: .travis.yml (hunk 2 above)
Comment (130s) [inline .travis.yml]: It's ok, I'd rather want to receive failure ;)

**Pull request closed without merging**